src/transaction/log_applier.c

```diff
@@ -7043,142 +7043,177 @@ la_print_log_arv_header (const char *database_name, LOG_ARV_HEADER * hdr, bool v
 }
 
 /*
- * la_log_page_check() - test the transaction log
- *   return: void
- *   log_path: log path
- *   page_num: test page number
+ * la_get_applied_log_info() - 
```

Comment:
why don't you add a summary about the function here? it would be good for maintenance.

Reply:
Right, I will add it as soon as possible